### ISSUE TITLE
crash caused by RFTools dimension wiping

### ISSUE THREAD

**Issue by DKnightADRI** [AllTheMods/ATM-3#637]
## Expected Behavior
When executing the rftdim safedel command it should delete the dimension with the ID passed as a parameter.

## Current Behavior
It seems to a crash, but not every time. 

## Possible Solution
I wish I knew...

## Steps to Reproduce
1. Create a random rftools dimension
2. Enter it and leave again
3. Extract the dimension tab and safedel it.

## Log
https://gist.github.com/DKnightADRI/08f3fae5a0a550376f5b37eba68cd685

**Comment by Yoosk** [comment]
Most likely mod issue, however i would check if removing world by survival means works properly.

**Comment by DKnightADRI** [comment]
Alright, will try that soon

**Comment by DKnightADRI** [comment]
Yes. This also seems to happen sending over tnt to destroy a dimension.
